static/app/views/alerts/rules/details/body.tsx

```diff
@@ -158,48 +158,58 @@ export default class DetailsBody extends React.Component<Props> {
-        {this.getFilter()}
-
-        <SidebarHeading>{t('Conditions')}</SidebarHeading>
-        {criticalTrigger && this.renderTrigger(criticalTrigger)}
-        {warningTrigger && this.renderTrigger(warningTrigger)}
-
-        <SidebarHeading>{t('Other Details')}</SidebarHeading>
-        <KeyValueTable>
-          <Feature features={['organizations:team-alerts-ownership']}>
-            <KeyValueTableRow
-              keyName={t('Team')}
-              value={
-                teamActor ? (
-                  <ActorAvatar actor={teamActor} size={24} />
-                ) : (
-                  <IconUser size="20px" color="gray400" />
-                )
-              }
-            />
-          </Feature>
-
-          {rule.createdBy && (
-            <KeyValueTableRow
-              keyName={t('Created By')}
-              value={<CreatedBy>{rule.createdBy.name ?? '-'}</CreatedBy>}
-            />
-          )}
-
-          {rule.dateModified && (
-            <KeyValueTableRow
-              keyName={t('Last Modified')}
-              value={<TimeSince date={rule.dateModified} suffix={t('ago')} />}
-            />
-          )}
-        </KeyValueTable>
+        <SidebarGroup>
+          <SidebarHeading>{t('Metric')}</SidebarHeading>
+          <RuleText>{this.getMetricText()}</RuleText>
+        </SidebarGroup>
+
+        <SidebarGroup>
+          <SidebarHeading>{t('Environment')}</SidebarHeading>
+          <RuleText>{rule.environment ?? 'All'}</RuleText>
+        </SidebarGroup>
+
+        <SidebarGroup>
+          <SidebarHeading>{t('Filters')}</SidebarHeading>
+          {this.getFilter()}
+        </SidebarGroup>
+
+        <SidebarGroup>
+          <SidebarHeading>{t('Conditions')}</SidebarHeading>
+          {criticalTrigger && this.renderTrigger(criticalTrigger)}
+          {warningTrigger && this.renderTrigger(warningTrigger)}
+        </SidebarGroup>
+
+        <SidebarGroup>
+          <SidebarHeading>{t('Other Details')}</SidebarHeading>
+          <KeyValueTable>
+            <Feature features={['organizations:team-alerts-ownership']}>
+              <KeyValueTableRow
+                keyName={t('Team')}
+                value={
+                  teamActor ? (
+                    <ActorAvatar actor={teamActor} size={24} />
+                  ) : (
+                    <IconUser size="20px" color="gray400" />
+                  )
+                }
+              />
+            </Feature>
+
+            {rule.createdBy && (
+              <KeyValueTableRow
+                keyName={t('Created By')}
+                value={<CreatedBy>{rule.createdBy.name ?? '-'}</CreatedBy>}
+              />
+            )}
+
+            {rule.dateModified && (
+              <KeyValueTableRow
+                keyName={t('Last Modified')}
+                value={<TimeSince date={rule.dateModified} suffix={t('ago')} />}
+              />
+            )}
+          </KeyValueTable>
+        </SidebarGroup>
```

Comment:
This is just wrapping everything in a `<SidebarGroup>` which adds a bit of margin